### ISSUE TITLE
How could I access side-menu instance in the ContentView?

### ISSUE THREAD

**Issue by iostalk** [Kureev/react-native-side-menu#96]
Excuse me, how could I access side-menu instance in the ContentView?

**Comment by Kureev** [comment]
Sure, you can get an access to `menuActions` on first-level child of ContentView as prop! See https://github.com/Kureev/react-native-side-menu/blob/master/examples/simple/index.ios.js#L54

**Comment by iostalk** [comment]
```jsx
var MyV2EXApp = React.createClass({
  render: function() {
    var menu = <LeftMenu navigator={navigator}/>;
    return (
      <SideMenu ref="sidemenu" touchToClose={true} menu={menu}>
        <NavigatorIOS
          style={styles.navContainer}
          initialRoute={{
            component: TopicList,
            title: '最新'
          }}
        />
      </SideMenu>
    );
  }
});
```
In the TopicList component, I find this.props.menuAction is empty. 
My react-native version is 0.10.0, maybe the React.cloneElement API has changed?

**Comment by iostalk** [comment]
It seems that caused by the NavigatorIOS nested, is any good ideas to solve this?

**Comment by iostalk** [comment]
In my first view , I want to enable side-menu's gesture. When enter a second view through navigation, I want to disable the slide-menus's gesture. That's what I want to implement.Could you give out an example?

**Comment by Kureev** [comment]
Okay, I can explain why it happens: in your case `NavigatorIOS` is a ContentView to which I apply `menuActions` prop. I think the only one way to solve this issue is to use `context` for this purpose. I'll try to fix it on this weekend

**Comment by iostalk** [comment]
Maybe I haven't understood context yet, how could it fix this?

**Comment by iostalk** [comment]
I have an idea, could I listen to the navigation's change event, In the callback function, Due to the route info, then disable the side menu.
Is it possible?

**Comment by Kureev** [comment]
I can't pass `menuActions` thru the props deeper, than for first level children. Context could be applied to any nested-level children.

**Comment by iostalk** [comment]
My requirement is like this:
In the first view, the side-menu guesture is enabled, and after the second pushed, the slide-menu guesture is disabled.When pop to the first view , side-menu guesture becomes enabled again.
So I think need more than access menuAction in anywhere, but I need some way to listen to the navigation change, didn't I?

**Comment by Kureev** [comment]
But you always has control over the moment when you pop second view, so why can't you just pop the view and then disable gestures?

**Comment by iostalk** [comment]
Err... When entering the second view, I can disable gestures, but when the user press the left button on the navigation bar, I need to listen to the back event on the second view, and reenable the gestures, it is strange , isn't it?
I think there must be a method to listen to the navigator change globally.

**Comment by Kureev** [comment]
Solved in `0.14.0`. Check the [release notes](https://github.com/Kureev/react-native-side-menu/releases/tag/v0.14.0).
